### ISSUE TITLE
Randomize ability usage time/%.

### ISSUE THREAD

**Issue by cdmichaelb** [ensingm2/SteamMonsterGameScript#119]
Often times you'll see a spawner die instantly but 30+ people used cripple spawner on it. I suggest creating a random range such as 90-100. There will still be overlap, but perhaps it'll make usage more efficient over time. Same with napalm and nuke and clusterbomb.

Maybe have it a  "High activity" toggle.

Not sure if this is possible but maybe create an irc channel for automatically coordinating abilities via the script.

**Comment by ensingm2** [comment]
Yeah there should be some kind of %chance for those abilities. Maybe have a chance to use that's `Math.random()  * some_const / players_in_lane` in an attempt to avoid this? Probably only really need it for the cripple spawner/monster items.

> Not sure if this is possible but maybe create an irc channel for automatically coordinating abilities via the script.

I feel like creating an automated irc channel for bot coordination is skirting the lines of exploiting the IRC client/server without their consent.

**Comment by thelulzy** [comment]
I hate to parrot information, but someone was saying that cripples were "refunded" if they were used at a time when they would have no effect. Not sure of the truth in that.

**Comment by cdmichaelb** [comment]
This is a little complex, but perhaps use IRC simply as a way to "Check-in" that the script is being used. Then compare checked'd-in clients against a list of available consumables similar to what http://drx.pl/steamgame/ranking_41672.html does for Wormholes. Then using some calculation (that ever script should know and agree with without having to talk to each other) calculate which clients should use an item or nuke, etc based on guesstimated hp (assuming nuke, etc, can be checked in a similar way to that url)

**Comment by vanZeben** [comment]
For the scope of this mini-game (A game that ends in 5 days time), I personally see that as a huge time sync for something like this... If there was actually a reward for being number 1, or if this wasn't simply a 2 week event, I would feel differently, but that's just me

**Comment by ensingm2** [comment]
Agreed.
